build-ps/percona-server.spec

```diff
@@ -708,6 +712,7 @@ fi
 %attr(644, root, root) %{_datadir}/percona-server/uninstall_rewriter.sql
 %attr(644, root, root) %{_datadir}/percona-server/magic
 %if 0%{?systemd}
+%attr(644, root, root) %{_unitdir}/mysqld@.service
```

Comment:
05:40:00 error: Installed (but unpackaged) file(s) found:
05:40:00    /usr/lib/systemd/system/mysqld@.service

So here it is